code/preprocessing/stopword_remover.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Oct  9 12:00:44 2021
+
+@author: lschiesser
+"""
+
+from code.preprocessing.preprocessor import Preprocessor
+from code.util import TWEET_TOKENIZED, COLUMN_STOPWORDS
+from nltk.corpus import stopwords
+import nltk
```

Comment:
```suggestion

```

Reply:
I suppose one doesn't have to import nltk separately because it is not used beyond the stopwords no?